src/Microsoft.AspNetCore.Routing/Tree/TreeRouter.cs

```diff
@@ -21,10 +22,6 @@ namespace Microsoft.AspNetCore.Routing.Tree
     /// </summary>
     public class TreeRouter : IRouter
     {
-        // Key used by routing and action selection to match an attribute route entry to a
-        // group of action descriptors.
-        public static readonly string RouteGroupKey = "!__route_group";
-
```

Comment:
Why is this safe to delete?

Reply:
OH! - I deleted it from the wrong spot. Will fix.

Reply:
It wasn't being used in TreeRouter anyway though. Perhaps it can stay deleted

Reply:
it's public in a public class, it needs to stay.